operator/model/src/main/java/io/apicurio/registry/operator/api/v1/spec/AppSpec.java

```diff
@@ -42,6 +42,16 @@ Configure features of the Apicurio Registry backend (app).
     @JsonSetter(nulls = SKIP)
     private AppFeaturesSpec features;
 
+    /**
+     * Configure features of the Apicurio Registry application.
+     */
+    @JsonProperty("auth")
+    @JsonPropertyDescription("""
+            Configure authentication and authorization of Apicurio Registry.
+            """)
+    @JsonSetter(nulls = SKIP)
+    private AppAuthSpec appAuthSpec;
```

Comment:
Let's keep the field name the same as property name.

Reply:
+1